src/components/custom/_non-eu-redirect-popup.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex } from 'components/containers'
+import { Button } from 'components/form'
+import { Text } from 'components/elements'
+import { Localize, localize } from 'components/localization'
+import device from 'themes/device'
+import { deriv_com_url } from 'common/constants'
+import { PopupModalTypes } from 'types/generics'
+
+const Background = styled.div`
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    background-color: var(--color-black);
+    opacity: 0.4;
+`
+const ModalWrapper = styled.div`
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    z-index: 200;
+`
+const ModalCard = styled.div`
+    position: relative;
+    z-index: 210;
+    display: flex;
+    flex-direction: column;
+    border-radius: 6px;
+    box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.3);
+    width: 384px;
+    height: 172px;
+    max-width: 44rem;
+    background: var(--color-white);
+    padding: 24px;
+
+    @media ${device.mobileL} {
+        width: 90%;
+        height: auto;
+    }
+`
+
+const ButtonWrapper = styled(Flex)`
+    justify-content: flex-end;
+    margin-top: 20px;
+`
+
+const StyledText = styled.div`
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+`
+
+const StyledButtonText = styled(Text)`
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 20px;
+
+    /* identical to box height, or 143% */
+
+    text-align: center;
+
+    /* dark/1 - prominent (text) */
+
+    color: #ffffff;
+
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+`
+const RedirectNonEuLogin = () => {
+    window.location.href = `https://${deriv_com_url}`
+}
+const NonEuRedirectPopUp = ({ is_open, setShowNonEuPopup }: PopupModalTypes) => {
+    const toggle = () => setShowNonEuPopup(!is_open)
+    return (
+        is_open && (
+            <>
```

Comment:
Do we need this?

Reply:
When the user clicks outside of the card we toggle the `is_open` and hide it. Therefor user has a way to cancel the redirection if he wants to stay in eu.deriv.com.

Reply:
I mean the extra fragment `<>`